ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -238,8 +251,15 @@ def create_storage_cluster(self):
         Review and Create storage cluster
 
         """
-        logger.info("Create on Review and create page")
-        self.do_click(locator=self.dep_loc["create_on_review"], enable_screenshot=True)
+        logger.info("Create storage cluster on 'Review and create' page")
+        if self.operator is OCS_OPERATOR:
+            self.do_click(
+                locator=self.dep_loc["create_on_review"], enable_screenshot=True
+            )
+        elif self.operator is ODF_OPERATOR:
+            self.do_click(
+                locator=self.dep_loc["create_storage_system"], enable_screenshot=True
+            )
         logger.info("Sleep 10 second after click on 'create storage cluster'")
         time.sleep(10)
```

Comment:
Also, after storagesystem is being created in 4.9, we can add several steps to check if successful navigation takes place to ODF tab under Storage or not. There are so many bugs around this and we might need to make sure this doesn't happen in regression runs.

Reply:
I dont think we need to check this on deployment test.. We can add these tests on tier1..